Automattic/wp-calypso
# Allow controls & upsells for SEO tools, Google Analytics, WordAds, and Search

#### Changes proposed in this Pull Request
 See title

<img width="767" alt="Screen Shot 2020-09-17 at 3 30 00 PM" src="https://user-images.githubusercontent.com/2810519/93534830-af03dd80-f8fa-11ea-8ee5-a8ce4a9ebfcc.png">
<img width="1079" alt="Screen Shot 2020-09-17 at 3 29 44 PM" src="https://user-images.githubusercontent.com/2810519/93534842-b1fece00-f8fa-11ea-8e66-69151e6a7c21.png">
<img width="1071" alt="Screen Shot 2020-09-17 at 3 29 48 PM" src="https://user-images.githubusercontent.com/2810519/93534855-b4612800-f8fa-11ea-92eb-3191ff891d39.png">


#### Testing instructions

1. Check `/marketing/tools/:site` with a free Jetpack plan
2. Verify the upsells appear
3. Check `/marketing/tools/:site` with a Security Jetpack plan
4. Verify Settings appear
5. Check `/settings/performance/:site` with a non-Complete Jetpack plan
6. Verify Upsell appears
7. Check `/settings/performance/:site` with a Complete Jetpack plan 
8. Verify Setting control appears
